src/pages/home/report/ReportActionsList.js

```diff
@@ -273,6 +281,7 @@ function ReportActionsList({
         return Math.ceil(availableHeight / minimumReportActionHeight);
     }, [windowHeight]);
 
+    const lastReportAction = useMemo(() => _.last(sortedReportActions) || {}, [sortedReportActions]);
```

Comment:
This seems randomly placed here, I would move it closer to where this is being used (basically where it was defined before)

Reply:
Ideally, we need to place them on the top of the list before `useCallback` and `useEffect,` but I didn't make it out of the scope of this PR. I will return it to where it is being used, but we also need to add a rule to the checklist with that subsequence.